spaces/domains.json

```
@@ -363,4 +363,5 @@
   "vote.faraland.io": "faradao.eth",
   "vote.nft-gods.com": "nft-gods.eth",
   "vote.sunrisegaming-dao.com": "sunrisegaming.eth"
+  "dao.babyspacefloki.com":"babyspacefloki.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.sunrisegaming-dao.com": "sunrisegaming.eth",
  "dao.babyspacefloki.com": "babyspacefloki.eth"
```